spec/raven/configuration_spec.rb

```diff
@@ -130,4 +130,16 @@
     end
   end
 
+  context 'configuration for sanitize fields' do
+    it 'should union default sanitize fields with user-defined sanitize fields' do
+      fields = Raven::Processor::SanitizeData::DEFAULT_FIELDS | %w(test monkeybutt)
```

Comment:
Shouldn't we use a field like "password" or something? Neither of the fields you're adding here are in DEFAULT_FIELDS